loops.js

```diff
@@ -71,3 +68,15 @@ for (let i = 0; i < 15; i++) {
 // >> 9
 // ==========================================
 
+for (let i = 0; i < 10; i++) {
+    if (i <= 2) {
+        console.log(i);
+    }
+    else if (i === 3 || i === 4 || i === 5) {
+        console.log("> " + i);
+    }
+    else if (i === 6 || i === 7 || i === 8 || i === 9) {
+        console.log(">> " + i);
+    }
+}
+
```

Comment:
Je code bij de bonusopdracht is goed geschreven. Hij doet het en de uitkomst is juist, maar hij is veel te lang geschreven omdat je nu elk nummer apart gaat aanroepen. Dit lukt je nooit als je dit bij 10.000 nummers moet doen. De code kun je het beter op deze manier schrijven:
`for (let i = 0; i <= 9; i++) {
    if (i <= 2) {
        console.log(i)
    } else if (i <= 5) {
        console.log(">" + i)
    } else {
        console.log(">>" + i)
    }
}`